core/storage/user/gae_models.py

```diff
@@ -914,8 +914,12 @@ class UserSubscriptionsModel(base_models.BaseModel):
 
     # IDs of activities (e.g., explorations) that this user subscribes to.
     # TODO(#10727): Rename this to exploration_ids and perform a migration.
+    # DEPRECATED. Do not use. Use exploration_ids instead.
     activity_ids = (
         datastore_services.StringProperty(repeated=True, indexed=True))
```

Comment:
Move this down below the `feedback_thread_ids`. The comment should be `DEPRECATED in v3.0.7. Do not use. Use exploration_ids instead.`

Reply:
Also, after this PR is created and in next (January) release, you should create a PR that will remove the deprecated field altogether.

Reply:
Done.